packages/atomic/src/components/atomic-breadcrumb-manager/atomic-breadcrumb-manager.tsx

```diff
@@ -187,13 +189,15 @@ export class AtomicBreadcrumbManager {
 
   private getCategoryBreadrumbValues(values: CategoryFacetBreadcrumb) {
     const breadcrumbsToShow = this.categoryCollapsedBreadcrumbsHandler(values);
+    const breadcrumbAriaLabel = breadcrumbsToShow.join('/');
```

Comment:
Does a screen-reader correctly read a joined path of strings?

Reply:
It reads it fine. 

But something I wasn't sure about it that the `categorybreadcrumb` are separated  buttons ([See example](https://user-images.githubusercontent.com/71234559/100274346-448ba100-2f2c-11eb-8b8f-4f80b875b669.png)). 
So, using that structure, I had two possible options:
1. Make `aria-label` the single path value, but it would then read "Remove inclusion filter on North America" for the first button, "Remove inclusion filter on United States" for the second... I don't think this was a good option because when you clicked on any of the path button, it removes everything and the user would end up confused. 

2. Make `aria-label` the whole path on every button (currently implemented). I chose that one because it tells the user what will be removed if any path button is clicked. 


Theres was an other solution to render the `categorybreadcrumb`. Instead of having 1 button per path value, we could have a single button. Its text would be formatted like 
```
<li>
  <button>
    breadcrumb.path.map(value => value.value).join(this.divider) //this.divider could be a prop for more customization
  </button>
</li>
```

Using this option, the `aria-label` would represent the button. Plus, it would make it easier to make the `x` clickable (PR comment above)

It looks like this:
![image](https://user-images.githubusercontent.com/71234559/100618147-6ea4e080-32e9-11eb-8d05-deaeb396b9d6.png)

Reply:
Yes make it a single button, in the case where we can deselect a partial path, we would have multiple buttons